embulk-output-jdbc/build.gradle

```diff
@@ -6,3 +6,42 @@ embulkPlugin {
     category = "output"
     type = "jdbc"
 }
+
+publishing {
+    publications {
+        maven(MavenPublication) {
+            pom {  // https://central.sonatype.org/pages/requirements.html
+                developers {
+                    developer {
+                        name = "Sadayuki Furuhashi"
+                        email = "frsyuki@gmail.com"
+                    }
+                    developer {
+                        name = "Hitoshi Tanaka"
+                        email = "thitoshi@cac.co.jp"
+                    }
+                    developer {
+                        name = "takumakanari"
+                        email = "chemtrails.t@gmail.com"
+                    }
+                    developer {
+                        name = "Tomohiro Hashidate"
+                        email = "kakyoin.hierophant@gmail.com"
+                    }
+                    developer {
+                        name = "Megumi Tomita"
+                        email = "tomykaira@gmail.com"
+                    }
```

Comment:
@tomykaira Hi!

As written in the pull-request description, we're trying to nominate developers who have contributed a important portion in `embulk-output-jdbc/mysql/postgresql/redshift/sqlserver`.

We wonder if you're okay to nominate you in `pom.xml` of `embulk-output-jdbc/mysql/postgresql/redshift/sqlserver` on Maven Central, from your work in #115. What do you think?

I picked up your name and email from the Git commit log and/or your public GitHub user page. Please let me know if you don't want, or if you want to be listed in another name/email.